EUROPACE API Calls.postman_collection.json

```
@@ -1,10 +1,109 @@
 {
 	"info": {
-		"_postman_id": "e0d25fa8-01ba-44bc-b878-d56b6dbfa5fa",
+		"_postman_id": "22a6b062-f2d1-43f6-8494-9f0a6f77c555",
 		"name": "EUROPACE API Calls",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
+		{
+			"name": "BaufiSmart Kundenangaben API",
+			"item": [
+				{
+					"name": "Kundenangaben validieren",
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{access_token}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "POST",
+						"header": [
+							{
+								"key": "X-traceId",
```

```suggestion
								"key": "X-TraceId",
```
Wir verwenden `X-TraceId` mit einem grossen T.